### PR TITLE
Meme updated to version 5.1.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MEME/MEME-5.1.1-foss-2019b-Perl-5.30.0-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'MEME'
+version = '5.1.1'
+versionsuffix = '-Perl-%(perlver)s-Python-%(pyver)s'
```
Comment (Micket) [inline easybuild/easyconfigs/m/MEME/MEME-5.1.1-foss-2019b-Perl-5.30.0-Python-3.7.4.eb:9]: We have started to drop the superflous perl suffix, as we always stick to 1 version per toolchain anyway.
```suggestion
versionsuffix = '-Python-%(pyver)s'
```

File: easybuild/easyconfigs/m/MEME/MEME-5.1.1-intel-2019b-Perl-5.30.0-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'MEME'
+version = '5.1.1'
+versionsuffix = '-Perl-%(perlver)s-Python-%(pyver)s'
```
Comment (Micket) [inline easybuild/easyconfigs/m/MEME/MEME-5.1.1-intel-2019b-Perl-5.30.0-Python-3.7.4.eb:9]:
```suggestion
versionsuffix = '-Python-%(pyver)s'
```

File: easybuild/easyconfigs/m/MEME/MEME-5.1.1-foss-2019b-Perl-5.30.0-Python-3.7.4.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/m/MEME/MEME-5.1.1-foss-2019b-Perl-5.30.0-Python-3.7.4.eb]: Thanks, I was not aware of that. It is changed now.

File: easybuild/easyconfigs/m/MEME/MEME-5.1.1-intel-2019b-Perl-5.30.0-Python-3.7.4.eb (hunk 2 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/m/MEME/MEME-5.1.1-intel-2019b-Perl-5.30.0-Python-3.7.4.eb]: As above.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/222b004b01aea02f9a3bbb78afe8c9c6 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**